.database_consistency.todo.yml

```diff
@@ -0,0 +1,251 @@
+---
```

Comment:
This is like a Rubocop ToDo file. It is whitelisting current violations so we can remove them from here and fix them bit by bit.